content/millennium/r4/financial/financial-transaction.md

```diff
@@ -0,0 +1,206 @@
+* The financial-transaction-allocation.target field must reference an Encounter associated with an Account.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token&gt;', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+<%= definition_table(:financial_transaction, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST /Basic
+
+#### Body
+
+  <%= json(:R4_FINANCIAL_TRANSACTION_CASH) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: application/fhir+json
+Date: Mon, 04 Nov 2019 16:23:57 GMT
+Etag: W/""
+Last-Modified: Mon, 04 Nov 2019 16:23:57 GMT
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Basic/FT-PL-14266754
+Server-Response-Time: 3890.363996
+Status: 201 Created
+Vary: Origin
+X-Request-Id: 1638e30e497b93ff4383b2ff0eaeea68
+X-Runtime: 3.890282
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+#### Request
+
+    POST
+
+#### Body
+
+  <%= json(:R4_FINANCIAL_TRANSACTION_CARD) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: application/fhir+json
+Date: Mon, 04 Nov 2019 16:23:59 GMT
+Etag: W/""
+Last-Modified: Mon, 04 Nov 2019 16:23:59 GMT
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Basic/FT-PL-74389581
+Server-Response-Time: 3890.363996
+Status: 201 Created
+Vary: Origin
+X-Request-Id: 1638e30e497b93ff4383b2ff0eaeea69
+X-Runtime: 3.890282
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+#### Request
+
+    POST
+
+#### Body
+
+  <%= json(:R4_FINANCIAL_TRANSACTION_CHECK) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: application/fhir+json
+Date: Mon, 04 Nov 2019 16:23:58 GMT
+Etag: W/""
+Last-Modified: Mon, 04 Nov 2019 16:23:58 GMT
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Basic/FT-PL-3335800133
+Server-Response-Time: 3890.363996
+Status: 201 Created
+Vary: Origin
+X-Request-Id: 1638e30e497b93ff4383b2ff0eaeea70
+X-Runtime: 3.890282
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+#### Request
+
+    POST
+
+#### Body
+
+  <%= json(:R4_FINANCIAL_TRANSACTION_ADJUSTMENT) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: application/fhir+json
+Date: Mon, 04 Nov 2019 16:23:58 GMT
+Etag: W/""
+Last-Modified: Mon, 04 Nov 2019 16:23:58 GMT
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Basic/FT-PL-3335800133
```

Comment:
This location needs to be updated

Reply:
Not sure what to update it to

Reply:
FT-AL

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/51acf7d28f90f2f7b2e079e47387bcac290bcb82#diff-26870e1b94fc9955e130f3f940dbe2e7R256

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/51acf7d28f90f2f7b2e079e47387bcac290bcb82#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR179

Reply:
+1